Sites/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+
+<html>
```

Comment:
Recommend keeping the `lang` attribute on your `html` element:
```suggestion
<html lang="en">
```